docs/team/chan-j-d.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: page
+title: Jun Da's Project Portfolio Page
```

Comment:
you might want to write your full name here i think it's what everyone else is doing

Reply:
good point, thanks